anime-flutter/anime_galaxy/lib/anime_explor_list/ui/widget/card_members/card_series.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:anime_galaxy/anime_explor_list/ui/widget/circular_image/circular_image.dart';
+import 'package:flutter/material.dart';
+
+import '../../../style_explore_list.dart';
+
+class CardMembers extends StatelessWidget {
+  String url_image;
```

Comment:
rename it to imageUrl cause lowerCamelCase  is the default naming convention in dart variable